src/validation/loader.rb

```diff
@@ -38,10 +38,11 @@ def initialize(metalware_config, cache_configure: false)
       end
 
       def configure_data
-        return @configure_data if @configure_data
-        tree = Validation::Configure.new(config, repo_configure_data).tree
-        @configure_data = tree if cache_configure
-        tree
+        if cache_configure
+          @configure_data ||= configure_data_tree
```

Comment:
I know this is the result of my original code, however what do you think about changing `configure_cache` to just `cache`?

It makes sense that if this is a class that `caches` it is an all or nothing approach. The default should still be not to `cache`. I'm happy for this change to occur in a latter PR if its not suited here.